src/main/scala/gestalt/api.scala

```diff
@@ -67,6 +68,11 @@ object api extends Toolbox {
     Ident.apply("java")
   }
 
+  def Ident(name: "_no_package_")(implicit dummy: Dummy2): TermTree = {
+    import options.unsafe
+    Ident.apply("<empty>")
+  }
+
```

Comment:
This is very nice. We can rename `_no_package_` to `_empty_`, WDYT?

Also, rebase and squash a little the commits history.